homeassistant/components/analytics_insights/strings.json

```diff
@@ -44,6 +44,12 @@
     "sensor": {
       "custom_integrations": {
         "name": "{custom_integration_domain} (custom)"
+      },
+      "total_active_installations": {
+        "name": "Total active installations"
+      },
+      "total_reports_integrations": {
+        "name": "Total reported integrations"
```

Comment:
Hmm, This is a strange name but I can't come up with something better here

Reply:
Yea, was also not overly joyed about it, but could not think of a better name.